index.rst

```diff
@@ -350,6 +347,22 @@ Markdown cells
 Every code cell should be preceded with a markdown cell that contains instructional statements
 written in the imperative mood (see the section on :ref:`narrative text <narrative-text>`).
 
+Use markdown format, and avoid ``html`` format.
+
+
+Line breaks
+^^^^^^^^^^^
+
+Avoid ``html`` format line breaks ``<br>`` in favor of two spaces at the end of the line.
```

Comment:
A `\` at the end of the line might work as well if the editors strip trailing white space.